robust_intelligence_ai_firewall/manifest.json

```
@@ -0,0 +1,66 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "1d208134-9005-4a79-bbc1-445950d1a5c7",
+  "app_id": "robust-intelligence-ai-firewall",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor AI Firewall Results using Datadog.",
+    "title": "Robust Intelligence AI Firewall",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Robust Intelligence AI Firewall",
+        "image_url": "images/ai-firewall.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Robust Intelligence AI Firewall Results dashboard",
+        "image_url": "images/firewall-dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::AI/ML",
+      "Offering::Integration",
+      "Submitted Data Type::Metrics"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "source_type_name": "Robust Intelligence AI Firewall",
+      "configuration": {
+        "spec": "assets/configuration/spec.yaml"
+      },
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "robust_intelligence_ai_firewall.",
+        "check": "",
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    },
+    "dashboards": {
+      "ai_firewall_results": "assets/dashboards/robust_intelligence_ai_firewall_overview.json"
+    },
+    "monitors": {},
+    "logs": {},
+    "saved_views": {}
+  },
+  "author": {
+    "support_email": "help@robustintelligence.com",
+    "name": "Mayank Rajoria",
```

```suggestion
    "name": "Robust Intelligence",
```
This should be the company name